src/App.tsx

```diff
@@ -1,10 +1,37 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoForm } from './components/TodoForm';
+import { TodoList } from './components/TodoList';
+import { TodoFooter } from './components/TodoFooter';
+import { ErrorMessage } from './components/ErrorMessage';
+import { Todo } from './types/Todo';
+import { client } from './utils/fetchClient';
+import { FilterOptions } from './types/FilterOptions';
 
-const USER_ID = 0;
+const USER_ID = 11272;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterOption, setFilterOption] = useState(FilterOptions.All);
+  const [hasError, setHasError] = useState(false);
+
+  const activeTodosCount = todos
+    .filter(todo => todo.completed === false)
+    .length;
+
+  useEffect(() => {
+    client.get<Todo[]>(`/todos?userId=${USER_ID}`)
```

Comment:
in project already exists function getTodos
https://github.com/mate-academy/react_todo-app-loading-todos/blob/master/src/api/todos.ts